crux-http-server/src/crux/http_server/query.clj

```diff
@@ -33,11 +33,20 @@
     :description "EDN formatted Datalog query"
     :decode/string (fn [_ q] (util/try-decode-edn q))}))
 
+;; unfortunately neither the inline spec conformer or st :decode/string are sufficient, therefore try-decode-edn coercion also takes place data-browser-query
+(s/def ::args
```

Comment:
Not sure I'd personally include a lot of this if it isn't really doing anything - could likely leave it as an `any?` spec for now, and leave the comment in case we go back to fix it later.

Reply:
Yep I agree it's not really necessary to merge this bit, but I felt I should include it during this review stage for completeness :)

Reply:
Is there a better process for discussing such snippets? e.g. just create another branch on my clone and reference it in the description?

Reply:
Hmmm - I'd probably just create a comment on the line itself and copy the snippet into that.

Reply:
assuming this is now used?

Reply:
yep :) since my latest force-push @ https://github.com/juxt/crux/pull/1507/commits/60c6fc387aef3bb05fe5322fa9bc0c6a8e12b8b4